internal/commands/pull/command.go

```diff
@@ -60,23 +62,47 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return err
 	}
 
-	pathTarget, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
+	clientZipPkgs, err := cmd.inputs.resolveClient(ui, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	pathRelative, err := filepath.Rel(profile.WorkingDirectory, pathTarget)
+	pathTarget, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
```

Comment:
maybe instead of `pathTarget` we can call this `pathBackend` now?  and maybe do one even better, we can have:

```go
pathApp, zipPkg, err := cmd.doExport(...)
pathBackend := pathApp // new reference allows us to keep things a bit more "immutable"/distinct

if clientZipPkgs != nil {
  pathFrontend := filepath.Join(pathApp, local.FrontendPath)

  // write zip files

  // now we update pathBackend, and pathApp can still remain accurate
  // if this line of code were to move for w/e reason, we don't risk creating a bad `pathFrontend` above
  pathBackend = filepath.Join(pathApp, local.BackendPath)
}
```